### PR TITLE
Commerce 7956 Import Task

### PR DESCRIPTION
**Pull request by upalatucci**
feat. @matijapetanjek 
cc: @igorbeslic, @ivicac

### CODE REVIEW

**Comment by upalatucci** [comment]
ci:test:sf

**Comment by upalatucci** [comment]
ci:forward

**Comment by matijapetanjek** [comment]
ci:stop

**Comment by ivicac** [comment]
ci:stop

**Comment by upalatucci** [comment]
ci:stop

**Comment by igorbeslic** [comment]
https://github.com/liferay-commerce/liferay-portal/pull/2045

**Comment by igorbeslic** [comment]
@upalatucci can you please close this PR. there is new PR with resolved conflicts:
https://github.com/liferay-commerce/liferay-portal/pull/2045

File: modules/apps/batch-planner/batch-planner-web/test/components/Import/ImportMappingItem.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+	it('must show dropdown items on select click', () => {
+		const {getByLabelText, getByText} = render(
+			<ImportMappingItem {...BASE_PROPS} />
+		);
+
+		act(() => {
+			fireEvent.click(getByLabelText(BASE_PROPS.field));
+		});
+
+		getByText(BASE_PROPS.selectableFields[0].label);
```
Comment (FabioDiegoMastrorilli) [inline modules/apps/batch-planner/batch-planner-web/test/components/Import/ImportMappingItem.js:57]: add an expect please

**Pull request closed without merging**